docs/ref/models/querysets.txt

```diff
@@ -1844,34 +1844,42 @@ they query the database each time they're called.
 .. method:: get(**kwargs)
 
 Returns the object matching the given lookup parameters, which should be in
-the format described in `Field lookups`_.
+the format described in `Field lookups`_. Normally you should use lookups
+that are guaranteed unique by the database, such as the primary key or fields
+in a unique constraint. For example::
 
-``get()`` raises :exc:`~django.core.exceptions.MultipleObjectsReturned` if more
-than one object was found. The
-:exc:`~django.core.exceptions.MultipleObjectsReturned` exception is an
-attribute of the model class.
+    Entry.objects.get(id=1)
 
-``get()`` raises a :exc:`~django.db.models.Model.DoesNotExist` exception if an
-object wasn't found for the given parameters. This exception is an attribute
-of the model class. Example::
+If ``get()`` finds more than one object, it raises a
+:exc:`Model.MultipleObjectsReturned <django.db.models.Model.MultipleObjectsReturned>`
+exception::
 
-    Entry.objects.get(id='foo') # raises Entry.DoesNotExist
+    Entry.objects.get(name='A Duplicated Name') # raises Entry.MultipleObjectsReturned
 
-The :exc:`~django.db.models.Model.DoesNotExist` exception inherits from
-:exc:`django.core.exceptions.ObjectDoesNotExist`, so you can target multiple
-:exc:`~django.db.models.Model.DoesNotExist` exceptions. Example::
+If ``get()`` doesn't find any object, it raises a
+:exc:`Model.DoesNotExist <django.db.models.Model.DoesNotExist>` exception::
+
+    Entry.objects.get(id='non-existent-id') # raises Entry.DoesNotExist
```

Comment:
I'd move the `Model.DoesNotExist` example above `Model.MultipleObjectsReturned` case as from my experience it is much more common.

Also, maybe something like `id=-999` instead of `id='non-existent-id'`? Usually `id` is going to be an `AutoField` and it feels better to use the correct data type.

Reply:
Both fair points!